Базовая стилизация всех страниц

Rodika94 opened the pull request:
что-то пулриквест сайт не находит

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2114175/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/2114175-sedona-34/2/

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2114175/tasks/6)

Rodika94 committed:
вернула список

до этого решила что лучше без списска, но стилизовать с оберткой все таки удобнее

Rodika94 committed:
исправила на один список

Rodika94 committed:
подправила читаемость

Rodika94 committed:
убрала дивы

убрала дивы так как они для крупных элементов а тут кода с гулькин писюн для дива

Rodika94 committed:
Update catalog.html

убрала все лишние стилевые классы отсебятные

Rodika94 committed:
тут тоже убрала лишние классы

вся классовая порнография убрана. код почти голый. все не смогла убрать ибо там надо на мой вгляд все таки

Rodika94 committed:
изменила хлебные крошки на упорядоченный список

Rodika94 committed:
добавила картинки в основную страницу

Rodika94 committed:
добавила в каталог картинки

Rodika94 committed:
подправила код по волидатору

добавила картинок

Rodika94 committed:
подправила читаемость и убрала лишние li

Dima-Dim commented on index.html:
```diff
@@ -52,46 +52,52 @@ <h1 class="visually-hidden">Декоративное фото</h1>
+                <h3 class="advantages-title">Там есть мост дьявола</h3>
+                <p>— №2 —</p>
+                <p>Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+              </div>>
```
`</div>>` одна `>` в конце лишняя.

Dima-Dim reviewed (changes requested):
Images/Amara-Resort.jpg и т.п. нарушает критерий https://up.htmlacademy.ru/htmlcss/34/criteries#PROJ-03

Images/index-background.jpg и Images/index-background.png для чего 2 одинаковых файла?

Rodika94 committed:
исправила название папки и пути

Rodika94 committed:
удалила пнг

Dima-Dim commented on catalog.html:
```diff
@@ -125,7 +125,7 @@ <h2 class="visually-hidden">Список товаров с фильтрами</h
           <ul class="product-list">
             <li class="product-card">
-              <img class="product-card-image" src="Images/Amara-Resort.jpg" width="300" height="211" alt="Amara Resort and Spa.">
+              <img class="product-card-image" src="images/Amara-Resort.jpg" width="300" height="211" alt="Amara Resort and Spa.">
```
Что на счёт https://up.htmlacademy.ru/htmlcss/34/criteries#PROJ-03 ? :) Всё ещё `Amara-Resort.jpg`.

Rodika94 committed:
испарвила название

Dima-Dim reviewed (changes requested):
Файлы всё так же названы с использование букв в верхнем регистре.
В опубликованой версии https://htmlacademy-htmlcss.github.io/2114175-sedona-34/2/ изображения не загружаются

Rodika94 committed:
мереместила из

Rodika94 committed:
добавила в

Rodika94 committed:
подключила шрифт

Rodika94 committed:
добавила правильно шрифты и немного стилей для body

Rodika94 committed:
исправила шапку

Rodika94 committed:
добавила класс для всех картинок

в основную страницу так же добавила классы только для картинок. общий класс для display: block;

Rodika94 committed:
сделала оформления шапок и приступаю к основной части контента

Rodika94 committed:
убрала пока что класс

Rodika94 committed:
немного стилей для проверки

Rodika94 committed:
все добавленные стили

добавляла с подписью кто есть кто

Rodika94 committed:
проверка

Dima-Dim commented on catalog.html:
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8"> 
     <title>Каталог поиска</title> 
+    <link href="https://fonts.googleapis.com/css2?family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
```
Вот это подключение шрифта со стороннего сервиса.

Dima-Dim commented on styles/styles.css:
```diff
@@ -0,0 +1,7 @@
+body {
+    font-family: "PT Sans";
+    src: url("../fonts/ptsans-400.woff2") format("woff2"),
```
А вот это из локального файла.
Нужто либо одно, либо другое.
